internal/admin/dashboard/templates/sidebar.html

```
@@ -0,0 +1,86 @@
+        <a href="/admin/dashboard/guardrails" class="nav-item" :class="{ active: page === 'guardrails' }" title="Guardrails (experimental)" x-show="guardrailsPageVisible()" @click.prevent="navigate('guardrails')">
+            <i data-lucide="shield-check" class="nav-icon" aria-hidden="true"></i>
+            <span>Guardrails (experimental)</span>
+        </a>
+        <a href="/admin/dashboard/settings" class="nav-item" :class="{ active: page === 'settings' }" title="Settings" @click.prevent="navigate('settings')">
+            <i data-lucide="settings" class="nav-icon" aria-hidden="true"></i>
+            <span>Settings</span>
+        </a>
+    </nav>
+    <div class="sidebar-footer">
+        <div class="theme-toggle">
+            <button class="theme-btn" :class="{ active: theme === 'light' }" @click="setTheme('light')" title="Light" tabindex="-1">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)